
main(){

  try{
    int n;
    int age = 2;
    print(n! * age);
  }catch(e){
    print('El error es el siguiente: ${e.toString()}');
  }
}